AirlineReservation/Services/Airline/AirlineService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using AirlineReservation.Models.Data;
+using AirlineReservation.Services.Database;
+using AutoMapper;
+using MongoDB.Driver;
+
+namespace AirlineReservation.Services.Airline
+{
+    public class AirlineService : IAirlineService
+    {
+        private IMapper _mapper;
+        private IDatabaseContext _databaseContext;
+
+        public AirlineService(IMapper mapper, IDatabaseContext databaseContext)
+        {
+            _mapper = mapper;
```

Comment:
are we using mapper here?

Reply:
Meant to use in upcoming methods. Removed for now.